ratelimits/README.md

```diff
@@ -0,0 +1,67 @@
+# Chainlink Data Provider Ratelimit Reference Package
+
+Package to update data provider rate limits and plans and to use as reference
+to prevent getting limited by or spamming providers.
+
+## adding a new provider
+
+Each provider is defined within limits.json as so
+
+```json
+"1forge":[
+    {
+      "rateLimit1h":208.33,
+      "tierName":"starter"
+    },
+    {
+      "rateLimit1h":4166.66,
+      "tierName":"premium"
+    },
+    {
+      "rateLimit1h":41666.66,
+      "tierName":"business"
+    },
+    {
+      "rateLimit1h":416666.66,
+      "tierName":"business+"
+    }
+  ], ...
+```
+- Each Object in the provider's array corresponds to a "tier" offered, in ascending order by price.
+- Valid rateLimit definitions are `rateLimit1s`, `rateLimit1m`, and `rateLimit1h`, corresponding to the closest time-based
+limit providers declare, 1 second, 1 minute, and 1 hour, respectively, currently using 10,000 /s to reflect unlimited. 
```

Comment:
Might be more intuitive to have undefined key/val pairs or `Inf` as unlimited?

Reply:
👍 opted for `-1`